test/quantization/core/test_quantized_module.py

```diff
@@ -889,6 +889,187 @@ def test_embedding_bag_api(self, num_embeddings, embedding_dim, num_offsets, set
+        loaded_qconv_module = type(dynamic_module)(
+            in_channels, out_channels, kernel_size, stride=stride, padding=padding,
+            dilation=dilation, groups=groups, bias=bias, padding_mode=padding_mode)
+        loaded_qconv_module.load_state_dict(loaded_dict)
+
+        self.assertTrue(dir(loaded_qconv_module) == dir(dynamic_module))
+        self.assertTrue(dynamic_module._get_name() == loaded_qconv_module._get_name())
+        self.assertTrue(hasattr(loaded_qconv_module, '_packed_params'))
+        self.assertTrue(hasattr(loaded_qconv_module, '_weight_bias'))
+
+        self.assertEqual(dynamic_module.weight(), loaded_qconv_module.weight())
+        if bias:
+            self.assertEqual(dynamic_module.bias(), loaded_qconv_module.bias())
+        self.assertEqual(dynamic_module.scale, loaded_qconv_module.scale)
+        self.assertEqual(dynamic_module.zero_point,
+                         loaded_qconv_module.zero_point)
+        Y_loaded = loaded_qconv_module(X_fp32, reduce_range)
+        np.testing.assert_array_almost_equal(
+            Y.numpy(), Y_loaded.numpy(), decimal=0)
+
+        # Test serialization
+        b = io.BytesIO()
+        torch.save(dynamic_module, b)
+        b.seek(0)
+        loaded_conv = torch.load(b)
+
+        self.assertEqual(loaded_conv.bias(), dynamic_module.bias())
+        self.assertEqual(loaded_conv.scale, dynamic_module.scale)
+        self.assertEqual(loaded_conv.zero_point,
+                         dynamic_module.zero_point)
+
+        # Test copy and deepcopy
+        copied_conv = copy.copy(dynamic_module)
+        self.assertEqual(copied_conv.bias(), dynamic_module.bias())
+        self.assertEqual(copied_conv.scale, dynamic_module.scale)
+        self.assertEqual(copied_conv.zero_point,
+                         dynamic_module.zero_point)
+        Y_copied = copied_conv(X_fp32, reduce_range)
+        np.testing.assert_array_almost_equal(
+            Y.numpy(), Y_copied.numpy(), decimal=0)
+
+        deepcopied_conv = copy.deepcopy(dynamic_module)
+        self.assertEqual(deepcopied_conv.bias(), dynamic_module.bias())
+        self.assertEqual(deepcopied_conv.scale, dynamic_module.scale)
+        self.assertEqual(deepcopied_conv.zero_point,
+                         dynamic_module.zero_point)
+        Y_deepcopied = copied_conv(X_fp32, reduce_range)
+        np.testing.assert_array_almost_equal(
+            Y.numpy(), Y_deepcopied.numpy(), decimal=0)
+
+        # need to fix this
+        # JIT testing
+        self.checkScriptable(
+            dynamic_module, [[X_dq]],
+            check_save_load=True)
+
+        # Test from_float
+        conv_module = dynamic_module._FLOAT_MODULE(in_channels, out_channels, kernel_size)
+        conv_module.qconfig = torch.ao.quantization.default_dynamic_qconfig  # type: ignore[assignment]
+        prepare_dynamic(conv_module)
+        conv_module(X_dq)
+        quantized_conv_module = dq_mod.from_float(conv_module)
```

Comment:
Is this dynamically quantized conv module? If so change names including conv_module one.

Also do you need to check for functional correctness?

Reply:
no its the unquantized conv module, this is testing the quantization process. dynamic_module, quantized_module and conv_module are the dynamically quantized, statically quantized, and fp32 modules respectively.